Formula/linthesia.rb

```diff
@@ -0,0 +1,19 @@
+class Linthesia < Formula
+  desc "Synthesia-like software for Linux"
+  homepage "http://linthesia.sourceforge.net/"
+  url "https://github.com/linthesia/linthesia/archive/refs/tags/0.8.0.tar.gz"
+  sha256 "0bbfeb30909fbd3a3b33bc45e8f7e75944ba6b2135a627d5729f4bda50540027"
+  license "GPL-2.0"
+
+  depends_on "gsettings-desktop-schemas" => :build
```

Comment:
You probably need at least a `depends_on "meson" => :build` and `depends_on "ninja" => :build` here.